keyboards/maartenwut/mx5160/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "mx5160", 
+    "url": "https://github.com/Maartenwut/mx-5160", 
+    "maintainer": "qmk", 
+    "width": 22, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```

As the PCB supports more than one physical layout, this layout macro should be named `LAYOUT_all`.

```suggestion
        "LAYOUT_all": {
```